sys/phydat/phydat_str.c

```diff
@@ -64,6 +64,8 @@ const char *phydat_unit_to_str(uint8_t unit)
         case UNIT_TEMP_K:   return "K";
         case UNIT_LUX:      return "lx";
         case UNIT_M:        return "m";
+        case UNIT_M2:       return "sqm";
+        case UNIT_M3:       return "cbm";
```

Comment:
> on the other hand, that would result in something like ksqm or kcbm which doesn't look that good - so it might even be better to do no scale2str in that cases at all?

Makes the usage of the API less intuitive IMHO. But how about using `m²` and `m³`, instead of `sqm` and `cqm`? Or do we try to stay ASCII?